src/workerd/io/trace.c++

```diff
@@ -259,6 +260,7 @@ void Trace::copyTo(rpc::Trace::Builder builder) {
     }
   }
 
+  builder.setSourcemapsAvailable(sourcemapsAvailable);
```

Comment:
let's remove sourcemapsAvailable from here, since we solved this problem downstream.